src/hub/models.py

```diff
@@ -138,3 +136,38 @@ class HubMembership(models.Model):
 
     created_date = models.DateTimeField(auto_now_add=True)
     updated_date = models.DateTimeField(auto_now=True)
+
+
+# HubV2
```

Comment:
We've been moving away from keeping all our models within `models.py`. We don't have a codified system, but if you check out some of our latest Django applications (researchhub_comment, citation_entry), you can see how we created a folder named `related_models` and created separate files for each model.

Reply:
Done